src/hooks/useFacilityModelVersions.tsx

```diff
@@ -2,21 +2,28 @@ import { useCallback, useEffect, useState } from "react";
 
 import { getFacilityModelVersions } from "../database";
 import { Facility, ModelInputs } from "../page-multi-facility/types";
+import useError from "./useError";
 
 const useFacilityModelVersions = (facility: Facility | undefined) => {
   const [facilityModelVersions, setFacilityModelVersions] = useState<
     ModelInputs[] | undefined
   >([]);
 
+  const rethrowSync = useError();
+
   const updateModelVersions = useCallback(async () => {
     if (!facility) return;
-    const modelVersions = await getFacilityModelVersions({
-      facilityId: facility.id,
-      scenarioId: facility.scenarioId,
-      distinctByObservedAt: true,
-    });
-    setFacilityModelVersions(modelVersions);
-  }, [setFacilityModelVersions, facility]);
+    try {
+      const modelVersions = await getFacilityModelVersions({
+        facilityId: facility.id,
+        scenarioId: facility.scenarioId,
+        distinctByObservedAt: true,
+      });
+      setFacilityModelVersions(modelVersions);
+    } catch (e) {
+      rethrowSync(e);
+    }
+  }, [facility, rethrowSync]);
```

Comment:
Do we no longer need `setFacilityModelVersions` in this array?

Reply:
we don't, the functions returned by `useState` are guaranteed to have stable references and don't need to be included here